networkml/featurizers/funcs/host.py

```diff
@@ -169,373 +166,868 @@ def _lowest_ip_proto_ports(self, rows, ip_proto):
+        return self._tshark_25q_frame_len_in(rows)
+
+    def host_tshark_25q_frame_len_out(self, rows):
+        return self._tshark_25q_frame_len_out(rows)
+
+    def host_tshark_75q_frame_len_in(self, rows):
+        return self._tshark_75q_frame_len_in(rows)
+
+    def host_tshark_75q_frame_len_out(self, rows):
+        return self._tshark_75q_frame_len_out(rows)
+
+    def host_tshark_median_frame_len_in(self, rows):
+        return self._tshark_median_frame_len_in(rows)
+
+    def host_tshark_median_frame_len_out(self, rows):
+        return self._tshark_median_frame_len_out(rows)
+
+    def host_tshark_variance_frame_len_in(self, rows):
+        return self._tshark_variance_frame_len_in(rows)
+
+    def host_tshark_variance_frame_len_out(self, rows):
+        return self._tshark_variance_frame_len_out(rows)
+
+    def host_tshark_max_frame_len_in(self, rows):
+        return self._tshark_max_frame_len_in(rows)
+
+    def host_tshark_max_frame_len_out(self, rows):
+        return self._tshark_max_frame_len_out(rows)
+
+    def host_tshark_min_frame_len_in(self, rows):
+        return self._tshark_min_frame_len_in(rows)
+
+    def host_tshark_min_frame_len_out(self, rows):
+        return self._tshark_min_frame_len_out(rows)
+
+    def host_tshark_min_time_delta_in(self, rows):
+        return self._tshark_min_time_delta_in(rows)
+
+    def host_tshark_min_time_delta_out(self, rows):
+        return self._tshark_min_time_delta_out(rows)
+
+    def host_tshark_25q_time_delta_in(self, rows):
+        return self._tshark_25q_time_delta_in(rows)
+
+    def host_tshark_25q_time_delta_out(self, rows):
+        return self._tshark_25q_time_delta_out(rows)
+
+    def host_tshark_median_time_delta_in(self, rows):
+        return self._tshark_median_time_delta_in(rows)
+
+    def host_tshark_median_time_delta_out(self, rows):
+        return self._tshark_median_time_delta_out(rows)
+
+    def host_tshark_average_time_delta_in(self, rows):
+        return self._tshark_average_time_delta_in(rows)
+
+    def host_tshark_average_time_delta_out(self, rows):
+        return self._tshark_average_time_delta_out(rows)
+
+    def host_tshark_75q_time_delta_in(self, rows):
+        return self._tshark_75q_time_delta_in(rows)
+
+    def host_tshark_75q_time_delta_out(self, rows):
+        return self._tshark_75q_time_delta_out(rows)
+
+    def host_tshark_max_time_delta_in(self, rows):
+        return self._tshark_max_time_delta_in(rows)
+
+    def host_tshark_max_time_delta_out(self, rows):
+        return self._tshark_max_time_delta_out(rows)
+
+    def host_tshark_variance_time_delta_in(self, rows):
+        return self._tshark_variance_time_delta_in(rows)
+
+    def host_tshark_variance_time_delta_out(self, rows):
+        return self._tshark_variance_time_delta_out(rows)
+
+
+class SessionHost(HostBase, Features):
+
+    def _row_keys(self, row):
+        eth_src = row['eth.src']
+        eth_dst = row['eth.dst']
```

Comment:
These need to be checked, perhaps `row.get('eth.src', None)` and `row.get('eth.dst', None)`?